src/client/components/App.tsx

```
@@ -101,6 +101,59 @@ export default function App() {
     });
   };
 
+  // @ts-expect-error unused function
+  const transcribeInputAudio = async (browserAudioUrl: string) => {
```

ESLint

[error] 'transcribeInputAudio' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)